src/main/kotlin/com/arkivanov/composnake/Main.kt

```diff
@@ -15,38 +11,55 @@ import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.ui.Alignment
+import androidx.compose.ui.ExperimentalComposeUiApi
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.focus.FocusRequester
-import androidx.compose.ui.focus.focusModifier
 import androidx.compose.ui.focus.focusRequester
+import androidx.compose.ui.focus.focusTarget
 import androidx.compose.ui.input.key.Key
 import androidx.compose.ui.input.key.key
 import androidx.compose.ui.input.key.onKeyEvent
-import androidx.compose.ui.unit.IntSize
+import androidx.compose.ui.unit.DpSize
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Window
+import androidx.compose.ui.window.WindowState
+import androidx.compose.ui.window.application
 import kotlinx.coroutines.delay
 import kotlinx.coroutines.isActive
 
-fun main() {
+// These provide margins between the main box and the outer window.
+val ROW_PADDING = 8.dp
+val COL_PADDING = ROW_PADDING
+
+// These make the cells an attractive size.
+val CELL_WIDTH = 24.dp
+val CELL_HEIGHT = CELL_WIDTH
+val CELL_MODIFIER = Modifier.width(CELL_WIDTH).height(CELL_HEIGHT)
```

Comment:
No need to add constants for modifiers. You can just write `Modifier.size(CELL_SIZE)` where needed.